circle.yml

```diff
@@ -9,6 +9,8 @@ general:
 machine:
   node:
     version: 4.2
+  ruby:
+    version: "2.0"
```

Comment:
I had to put them back.  The ci requires the ruby version to be a string.  @rahulreddy did 10 push-ups in atonement.